[IMP] website: create menu with cards for webdesign (#14453)
odoo/documentation: content/applications/websites/website/web_design.rst

:nosearch:
:show-content:
:hide-page-toc:

==========
Web design
==========

> Design your website using :doc:`Building blocks <../website/web_design/building_blocks>`, shape your
> :doc:`Themes <../website/web_design/themes>` with various options, structure and present content with
> :doc:`Elements <../website/web_design/elements>` and display or hide building blocks using
> :doc:`Visibility <../website/web_design/visibility>` options.

Reviewer: ```suggestion
Design your website using :doc:`building blocks <../website/web_design/building_blocks>`, customize its
:doc:`theme <../website/web_design/themes>` with various options, structure and present content with
:doc:`elements <../website/web_design/elements>`, and display or hide building blocks using
:doc:`visibility settings <../website/web_design/visibility>`.
```
- to remove the caps were necessary
- to limit repetitions (*options*)
- I know that's what we used on that specific doc page but I'm not a fan of *shape* in this contexte/in doc in genral

.. cards::

   .. card:: Building blocks
      :target: web_design/building_blocks
      :large:

      Design your website by dragging and dropping building blocks, then editing them to fit your
      content and layout needs.

   .. card:: General theme
      :target: web_design/themes
      :large:

>       Shape your website’s theme with various options for its colors, fonts, and layout.

Reviewer: ```suggestion
      Customize your website’s theme by adjusting its colors, fonts, and layout.
```
